app/web/features/search/SearchResultsList.tsx

```diff
@@ -83,163 +75,107 @@ const useStyles = makeStyles((theme) => ({
-          .filter((user): user is User.AsObject => !!user);
-
-        const setFilter = () => {
-          map.current &&
-            filterUsers(
-              map.current,
-              Object.keys(searchFilters.active).length > 0
-                ? resultUsers.map((user) => user.userId)
-                : null,
-              handleMapUserClick
-            );
-
-          if (bbox && bbox.join() !== "0,0,0,0") {
-            map.current?.fitBounds(bbox, {
-              maxZoom: selectedUserZoom,
-            });
-          }
-        };
-
-        if (map.current?.loaded()) {
-          setFilter();
-        } else {
-          map.current?.once("load", setFilter);
-        }
-      },
-    }
-  );
-  const isSearching = Object.keys(searchFilters.active).length !== 0;
+  const hasAtLeastOnePageResults = true;
 
   return (
     <Paper className={classes.mapResults}>
-      {error && <Alert severity="error">{error.message}</Alert>}
+      {error && <Alert severity="error">{error}</Alert>}
+
       <Hidden smDown>
         <SearchBox
-          searchFilters={searchFilters}
-          updateMapBoundingBox={updateMapBoundingBox}
+          searchType={searchType}
+          setSearchType={setSearchType}
+          locationResult={locationResult}
+          setLocationResult={setLocationResult}
+          setQueryName={setQueryName}
+          queryName={queryName}
         />
       </Hidden>
-      {isSearching ? (
-        isLoading ? (
-          <CircularProgress className={classes.baseMargin} />
-        ) : hasAtLeastOnePage(results, "resultsList") ? (
+
+      <>
+        {isLoading && <CircularProgress className={classes.baseMargin} />}
+
+        {!isLoading && !hasAtLeastOnePageResults && (
+          <TextBody className={classes.baseMargin}>
+            {t("search_result.no_user_result_message")}
+          </TextBody>
+        )}
+
+        {!isLoading && hasAtLeastOnePageResults && (
           <HorizontalScroller
             breakpoint="sm"
             className={classes.scroller}
-            isFetching={isFetching}
+            isFetching={isLoading}
             fetchNext={fetchNextPage}
-            hasMore={hasNextPage}
+            hasMore={hasNext}
           >
-            {results.pages
-              .flatMap((page) => page.resultsList)
-              .map((result) =>
-                result.user ? (
-                  <SearchResult
-                    id={`search-result-${result.user.userId}`}
-                    className={classes.searchResult}
-                    key={result.user.userId}
-                    user={result.user}
-                    onSelect={handleResultClick}
-                    highlight={result.user.userId === selectedResult}
-                  />
-                ) : null
-              )}
+            {results &&
+              results.pages
+                .flatMap((page) => page.resultsList)
+                .map((result) =>
+                  result.user ? (
```

Comment:
How about making this a named variable? Just a bit easier to read hat's happening that way.